### PR TITLE
Rnr 118 empty avatar circles

### PR DESCRIPTION
**Pull request by jack829**
## JIRA Link
https://scrychat.atlassian.net/browse/RNR-118

## Changes
* fetch user's fb profile picture every time they open the app, send this to the api to update in the database
* utilize this field rather than possibly outdated `scryer_thumbnail_pic` for rendering friends' thumbnails

## Other Dependencies
https://github.com/TryScry/api/pull/35

### CODE REVIEW

**Comment by dblanchard13** [comment]
Running into the following error when testing this out:

Screen Shot 2019-04-24 at 1 27 52 PM


Looks like it's a conversation that Taylor was the owner of, but removed himself from. Which is a bug since that really should remove all of use from the conversation.

Also, still seeing white circles for my chats:

Screen Shot 2019-04-24 at 1 32 19 PM


From a quick investigation, that looks like it's only happening for removed matches. So I'm guessing tinder added some more security around their hosted images to make sure you can't see images of someone who blocked you.

**Comment by jack829** [comment]
Yeah so the thing I fixed was facebook pictures not loading, not match pictures (which come from tinder). Though they might have a similar issue. I hadn't seen white circles for matches' pics before, though

**Comment by dblanchard13** [comment]
Yeah, looks like the ticket is meant to address all the pictures though. I'd say we should split out a bug ticket for the conversations not being deleted when the owner removes themselves from it. And then a story ticket for displaying something instead of a white circle for removed matches (could potentially be a tad involved).

For this ticket though, could you add some defensive logic to fallback to the scryer_thumbnail_pic if there's no wizard? Could also just wait to merge this until the bug is fixed.

**Comment by jack829** [comment]
Oh yeah I wasn't saying "fuck you this PR is good" just what I looked into. I'll keep working on this -> but yeah probably a separate bug for a wizard removing himself from the convo

**Comment by dblanchard13** [comment]
Gotcha gotcha

**Comment by jack829** [comment]
Ok, I set `scryer_thumbnail_pic` as the fallback for now. I think for tinder it's the same thing - if a match is removed _and_ the picture cdn is expired then it's blank. Don't have a great way around that at the moment so I say throw it over the fence to a new story?

File: src/store/user/reducer/user.js [hunk 1]
```diff
@@ -18,3 +18,15 @@ export function setUser(state, action) {
     tutorials,
   }
 }
+
+export function setUserFbUrl(state, action) {
+  const { url } = action
+  console.log('url ', url)
```
Comment (dblanchard13) [inline src/store/user/reducer/user.js]: Think we're good to remove this?

**Pull request merged**